client/components/sign-in.jsx

```diff
@@ -11,6 +11,19 @@ export default class SignIn extends React.Component {
     this.handleUsernameChange = this.handleUsernameChange.bind(this);
     this.handlePasswordChange = this.handlePasswordChange.bind(this);
     this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleDemoClick = this.handleDemoClick.bind(this);
+  }
+
+  handleDemoClick() {
+    const form = document.querySelector('form');
+    const nameInput = form.querySelector('[name="newUsername"]');
+    const passwordInput = form.querySelector('[name="newPassword"]');
+    nameInput.value = 'Bob Ross';
+    passwordInput.value = 'Password';
+    this.setState({
+      username: 'Bob Ross',
+      password: 'Password'
+    });
```

Comment:
Rather than query and manipulate the DOM, in React apps, we update our state and pass props to react elements.

If you pass a `value` prop to each of your form inputs, setting state should be enough.